pkgs/servers/jellyfin/default.nix

```diff
@@ -28,8 +44,8 @@ stdenv.mkDerivation rec {
 
     makeWrapper "${dotnet-sdk}/bin/dotnet" $out/bin/jellyfin \
       --prefix LD_LIBRARY_PATH : "${stdenv.lib.makeLibraryPath [
-        sqlite
-      ]}" \
+        sqlite fontconfig freetype stdenv.cc.cc.lib
```

Comment:
The issue isn't with the `dotnet` executable per-se, but with a bundled library which, because we download the pre-compiled version, isn't properly linked. Here's what I get when I do `ldd opt/jellyfin/runtimes/linux-x64/native/libSkiaSharp.so` from a pure nix-shell:

```
linux-vdso.so.1 (0x00007ffe6fd6b000)
libpthread.so.0 => /nix/store/681354n3k44r8z90m35hm8945vsp95h1-glibc-2.27/lib/libpthread.so.0 (0x00007f2208f48000)
libdl.so.2 => /nix/store/681354n3k44r8z90m35hm8945vsp95h1-glibc-2.27/lib/libdl.so.2 (0x00007f2208f43000)
libfreetype.so.6 => not found
libfontconfig.so.1 => not found
libstdc++.so.6 => not found
libm.so.6 => /nix/store/681354n3k44r8z90m35hm8945vsp95h1-glibc-2.27/lib/libm.so.6 (0x00007f2208dad000)
libgcc_s.so.1 => /nix/store/681354n3k44r8z90m35hm8945vsp95h1-glibc-2.27/lib/libgcc_s.so.1 (0x00007f2208b95000)
libc.so.6 => /nix/store/681354n3k44r8z90m35hm8945vsp95h1-glibc-2.27/lib/libc.so.6 (0x00007f22089df000)
/nix/store/681354n3k44r8z90m35hm8945vsp95h1-glibc-2.27/lib64/ld-linux-x86-64.so.2 (0x00007f220995c000)
```